ngxunprivinst.sh

```diff
@@ -329,7 +340,11 @@ extract() {
             exit 1
         fi
         TARGETVER=$($ABSPATH/usr/sbin/nginx -v 2>&1 | cut -d '(' -f 2 | cut -d ')' -f 1 | cut -d'-' -f 3 | tr -d 'r')
-        if [ $TARGETVER -ge 31 ]; then
+        if [ $TARGETVER -ge 33 ]; then
+            mv $TMPDIR/license.jwt $ABSPATH/etc/nginx/license.jwt
+            echo "mgmt { license_token $ABSPATH/etc/nginx/license.jwt; }" >> $ABSPATH/etc/nginx/nginx.conf
```

Comment:
In r33, the `uuid_file` directive was replaced by `report_state_file`. The path can remain the same as for uuid_file, I think, however, for filename we can use the default one - `nginx-mgmt-state`.

Reply:
Hmmm, do you mean we also need a `state_path` set here for R33+?

Reply:
Unfortunately, yes, this is another change that breaks similar scenarios. Actually, we could keep the `uuid_file` directive, but nginx will log a warning that this directive is deprecated. Starting from 33, `state_path` will indicate the path to the state file.